pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -72,13 +72,13 @@ import com.pubnub.api.v2.entities.ChannelGroup
 import com.pubnub.api.v2.entities.ChannelMetadata
 import com.pubnub.api.v2.entities.UserMetadata
 import com.pubnub.api.v2.subscriptions.Subscription
+import com.pubnub.api.v2.subscriptions.SubscriptionOptions
 import com.pubnub.api.v2.subscriptions.SubscriptionSet
 import com.pubnub.internal.BasePubNubImpl
 import com.pubnub.kmp.CustomObject
 import java.io.InputStream
 
-interface PubNub :
-    com.pubnub.kmp.PubNub,
+actual interface PubNub :
```

Comment:
Is there a need to have `actual` interface PubNub when there is `expect interface PubNub` ?

Reply:
yes, there must be an actual for each target

Reply:
In those examples there one interface(there is no actual version of expect interface)
https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-connect-to-apis.html#expected-and-actual-functions

https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-connect-to-apis.html#different-entry-points